src/scss/themes/_package.scss

```diff
@@ -0,0 +1,87 @@
+@mixin oTeaserPackage {
+	@include oTeaserInverse;
+	width: 100%;
+
+	.o-teaser__image-placeholder {
```

Comment:
Oh—I forgot to mention. It's the only way I've manage to get the image behind the content for the package teasers ... :D